digid_eherkenning/models/eherkenning.py

```diff
@@ -9,6 +9,56 @@
 from .base import BaseConfiguration
 
 
+def get_default_requested_attributes_eherkenning():
+    return [
+        {
+            "name": "urn:etoegang:1.11:attribute-represented:CompanyName",
+            "required": True,
+            "purpose_statements": {
+                "en": "For testing purposes.",
```

Comment:
Are these `purpose_statements` going to be always the same?This is from the example concerning eIDAS right?

Reply:
All these fields are configurable, but this is just for the default!